imports/plugins/core/versions/server/migrations/56_change_inventoryQuantity_to_inventoryInStock.js

```diff
@@ -0,0 +1,55 @@
+import { Migrations } from "meteor/percolate:migrations";
+import * as collections from "/lib/collections";
+
+Migrations.add({
+  version: 56,
+  up() {
+    collections.Products.update(
+      {
+        type: "simple"
+      },
+      {
+        $rename: { inventoryQuantity: "inventoryInStock" }
+      },
+      {
+        multi: true
+      }
+    );
+
+    collections.Products.update(
+      {
+        type: "variant"
+      },
+      {
+        $rename: { inventoryQuantity: "inventoryInStock" }
+      },
+      {
+        multi: true
+      }
+    );
```

Comment:
You could merge these into one update call with `{ type: { $in: ["simple", "variant"] }`